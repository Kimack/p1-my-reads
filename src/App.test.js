import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-15';
import BooksApp from './App';

configure({ adapter: new Adapter() });

it('renders without crashing', () => {
  const component = <BooksApp />;
  expect(shallow(component)).toMatchSnapshot();
});
