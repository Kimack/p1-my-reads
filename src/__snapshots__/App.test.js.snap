// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="app"
>
  <Component
    allowHTML={false}
    noAnimation={false}
    style={Object {}}
  />
  <Route
    path="/search"
    render={[Function]}
  />
  <Route
    exact={true}
    path="/"
    render={[Function]}
  />
</div>
`;
